src/client/VZSidebar/index.tsx

```diff
@@ -94,6 +100,48 @@ export const VZSidebar = ({
     handleDrop,
   } = useDragAndDrop();
 
+  function List(props) {
+    let state = useListState(props);
```

Comment:
The style of this codebase prefers `const` over `let`. Please prefer `const` where possible. Thanks!